doctr/utils/geometry.py

```diff
@@ -129,10 +133,41 @@ def rotate_abs_boxes(boxes: np.ndarray, angle: float, img_shape: Tuple[int, int]
     return rotated_boxes
 
 
+def remap_boxes(boxes: np.ndarray, orig_shape: Tuple[int, int], dest_shape: Tuple[int, int]) -> np.ndarray:
```

Comment:
So actually, I realized a week ago that this is an issue:
if the shapes are different and the rbox is in relative coords, this doesn't work
If the aspect ratio is different, simply rescaling doesn't work, and the angle may change :/

Reply:
Well I think it actually is:
- imagine first a squared image, with an orientation of 45° (box behind read in the direction of the top right corner)
- now you switch to relative coords the rotated bounding box
- now the image is resized to a shape of (100, 200)
- remapping this is a warping transformation, and the box would still be pointing to the top right corner which is now about ~26.6°

So it's actually a bit more complex than it sounds :upside_down_face: